GCC_2019_07_10/dbplyr lightning talk/02 simple example.R

```diff
@@ -7,26 +7,27 @@ sql <-
 
 # Do the extract using the usual dbGetQuery (from odbc)
 tic()
-simple_1 <- dbGetQuery(SMRA_connection, sql)
+simple_1_extract <- dbGetQuery(SMRA_connection, sql)
```

Comment:
since you gave new names to the vars, perhaps there's no need for _1 and _2? it could just be simple_query and simple_extract?
I'm saying this because I thought from this bit that 1 will be associated with **extract** and 2 with **query** but the code below showed me I was wrong. I wouldn't want any other code users to have that expectation from the notation.

Reply:
The idea was to create a connection (in the reader's mind) between the 'extract' and the 'query' whilst making a distinction.
Previously they didn't have the distinction and one overwrote the other which wasn't great since the 'query' bit is a different object type to the 'extract' bit